stable/prometheus-mongodb-exporter/templates/deployment.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: {{ include "prometheus-mongodb-exporter.fullname" . }}
+  labels:
+    app.kubernetes.io/name: {{ include "prometheus-mongodb-exporter.name" . }}
+    helm.sh/chart: {{ include "prometheus-mongodb-exporter.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "prometheus-mongodb-exporter.name" . }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+  template:
+    metadata:
+      annotations:
+{{ toYaml .Values.annotations | indent 8 }}
+      labels:
+        app.kubernetes.io/name: {{ include "prometheus-mongodb-exporter.name" . }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
+    spec:
+      containers:
+        - name: {{ .Chart.Name }}
+          env:
+            - name: MONGODB_URL
+              value: {{ .Values.config.datasource }}
+              {{- if .Values.config.auth }}
+            - name: HTTP_AUTH
+              value: {{ .Values.config.auth }}
+              {{- end}}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          ports:
+            - name: http
+              containerPort: 80
```

Comment:
Why port 80 is exposed? It looks like the default listening port for exporter is [9216](https://github.com/percona/mongodb_exporter/blob/61e655e36c5ce4215d1732ce769fb591b82aa7b5/mongodb_exporter.go#L46).